server/container/core/pom.xml

```diff
@@ -102,6 +102,11 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-server</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is this needed?

Reply:
![image](https://user-images.githubusercontent.com/81145350/170651704-a6884294-a838-4b39-a184-600f8fb9be81.png)

https://github.com/apache/james-project/pull/1017/commits/1eba0e43271725543ef5720cbe7945116686c2a8